docs/quick-starts/framework/flutter/_integration.mdx

```diff
@@ -1,5 +1,7 @@
-import redirectUriFigure from '../../assets/ios-redirect-uri.png';
+import postSignOutRedirectUriFigure from '../../assets/flutter-post-sign-out-redirect-uri.png';
+import redirectUriFigure from '../../assets/flutter-redirect-uri.png';
 import Checkpoint from '../../fragments/_checkpoint-test-your-application.md';
+import ConfigurePostSignOutRedirectUri from '../../fragments/_configure-post-sign-out-redirect-uri.mdx';
```

Comment:
Copy these new fragments to `_for-tutorial.mdx`, so that the generated "Build X with Y" tutorials can also have these updates